data/constructions/rail.tres

```diff
@@ -4,13 +4,15 @@
 
 [resource]
 script = ExtResource( 1 )
-id = "Rail"
```

Comment:
why is this being renamed?

Reply:
For consistency as its named Connector everywhere else in the code, name stays unchanged though so display is still Rail

Reply:
I see.
My thinking was it's a rail tile, but it's type is a connector.
Like, in case we would also add "Road" as an example, which might not be as efficient as Rails, but would also be the "Connector" type.

Reply:
Other than that the code looks good to me.
haven't checked it out locally yet though. But files look good.

Reply:
Good point, ill change it back then